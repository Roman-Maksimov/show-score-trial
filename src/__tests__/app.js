import React from 'react';
import App from '../app';
import renderer from 'react-test-renderer';

test('should render App', () => {
  const component = renderer.create(
    <App />,
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});