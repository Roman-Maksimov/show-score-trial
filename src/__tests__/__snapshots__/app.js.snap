// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App 1`] = `
<div
  className="main"
>
  <div
    className="field"
  >
    <div
      className="wrapper"
      style={
        Object {
          "paddingBottom": "50%",
        }
      }
    >
      <div
        className="content"
      >
        <div
          className="col"
        >
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
        </div>
        <div
          className="col"
        >
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
        </div>
        <div
          className="col"
        >
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
        </div>
        <div
          className="col"
        >
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
        </div>
        <div
          className="col"
        >
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
        </div>
        <div
          className="col"
        >
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
        </div>
        <div
          className="col"
        >
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
        </div>
        <div
          className="col"
        >
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
        </div>
        <div
          className="col"
        >
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
        </div>
        <div
          className="col"
        >
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
        </div>
        <div
          className="col"
        >
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
        </div>
        <div
          className="col"
        >
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="tile"
              onClick={[Function]}
            >
              <div
                className="bg"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="buttons"
    >
      <div
        className="buttonsPadding"
      />
      <button
        className="button button"
        onClick={[Function]}
      >
        <div
          className="inner"
        >
          RESET
        </div>
      </button>
      <div
        className="buttonsSpacing"
      />
      <button
        className="buttonAccent button"
        onClick={[Function]}
      >
        <div
          className="inner"
        >
          RUN
        </div>
      </button>
      <div
        className="buttonsPadding"
      />
    </div>
  </div>
</div>
`;
